CHANGE_LOG

```diff
@@ -19,6 +19,21 @@ Pull-Requests:
 * PR `#8411 <https://github.com/numba/numba/pull/8411>`_: update version support table for 0.56.1 (`esc <https://github.com/esc>`_)
 * PR `#8412 <https://github.com/numba/numba/pull/8412>`_: Create changelog for 0.56.1 (`Andre Masella <https://github.com/apmasell>`_)
 * PR `#8413 <https://github.com/numba/numba/pull/8413>`_: Fix Azure CI for NumPy 1.23 and use conda-forge scipy (`Siu Kwan Lam <https://github.com/sklam>`_)
+* PR `#8414 <https://github.com/numba/numba/pull/8413>`_: Hotfix for 0.56.2 (`Siu Kwan Lam <https://github.com/sklam>`_)
+
+Authors:
+
+* `Andre Masella <https://github.com/apmasell>`_
+* `esc <https://github.com/esc>`_
+* `Graham Markall <https://github.com/gmarkall>`_
+* `jamesobutler <https://github.com/jamesobutler>`_
+* `Siu Kwan Lam <https://github.com/sklam>`_
+* `stuartarchibald <https://github.com/stuartarchibald>`_
+
+Version 0.56.1 (NO RELEASE)
+----------------------------------
```

Comment:
```suggestion
Version 0.56.1 (NO RELEASE)
---------------------------
```
Underline line length doesn't match length of the thing it's underlining? Expect the CI doc build will pick this up.

Reply:
oh yes, fixed in e46a8d7cbe8116afed575d82c31a6a1ad6e703ce